### ISSUE TITLE
Whether batch processing is implemented

### ISSUE THREAD

**Issue by sshzhang** [HKUST-KnowComp/FMG#23]
Hi author:
  I  want to know whether batch processing is implemented in your experiment code.

**Comment by sshzhang** [comment]
sorry bother you again , Can you  explain  the specific  meaning  of the document  which in  "/yelp-50k/exp_split/1 " directory.

**Comment by hzhaoaf** [comment]
@sshzhang What do you mean by "batch processing"? For the files in "yelp-50K/exp_split/1", most of them are recording relations between entities in Yelp. You may check the meta-graphs in Figure 3 in [the KDD paper](http://www.cse.ust.hk/~hzhaoaf/data/kdd17-paper.pdf), and the filenames are created correspondingly.

**Comment by sshzhang** [comment]
Thank you for you reply ,  I  means if I need cope with big data, The batch_size can release the memory capacity .

**Comment by hzhaoaf** [comment]
@sshzhang No. This programs are not designed and optimized for very large-scale datasets, though I did implement the training module with block gradient descent, which can kind of improve the running speed with the help of numpy support.

**Comment by sshzhang** [comment]
Thanks a lot.
